README.rst

```diff
@@ -125,6 +129,9 @@ If you have more than one role available to you, you'll be prompted to
 choose the role from a list; otherwise, if your credentials are correct,
 you'll just see the AWS keys printed on stdout.
 
+If you're using Docker with a U2F security key (Yubikey or similar) you won't
```

Comment:
This sentence is awkward,  perhaps you meant to say `If you're using a U2F security key with your Google account you won't be ...`

Can you expand on what `Docker/Host isolation` mean?

Reply:
I agree on the sentence; that definitely could have been more clear. Changes coming shortly. 

Regarding the Docker/Host isolation, I'll put a small summary in the README.rst, but in more detail:
* The user will likely (not confirmed) need to set up USB passthrough to their VM (for Docker Machine / Docker Toolbox)
* The container will likely (not confirmed) need to have additional mounts and/or the privileged flag. 
* I have confirmed it _does not_ work as currently documented if you have a U2F key enabled on your Google Account and you run this using Docker.